Can not start a local game

#### Environment

* TripleA 1.9.0.0.7422
* Multiple operating systems
    * Windows 8 x86_64 (Oracle Java 1.8.0_121)
    * Fedora Linux 26 (OpenJDK 1.8.0_151 and Oralce Java 1.8.0_144)

#### Reproduction steps

1. Start a local game.

#### Expected behavior

The game should start.

#### Actual behavior

The game appears to launch (i.e. loading dialog is displayed) but then immediately returns to the main menu.  The following error is displayed on stdout:

```
java.lang.reflect.InvocationTargetException
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1321)
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1296)
	at javax.swing.SwingUtilities.invokeAndWait(SwingUtilities.java:1348)
	at games.strategy.triplea.TripleA.startGame(TripleA.java:122)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:49)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$170(AbstractLauncher.java:51)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at org.pushingpixels.substance.api.ComponentState.getState(ComponentState.java:504)
	at org.pushingpixels.substance.api.ComponentState.getState(ComponentState.java:468)
	at org.pushingpixels.substance.internal.animation.StateTransitionTracker.setModel(StateTransitionTracker.java:291)
	at org.pushingpixels.substance.internal.ui.SubstanceCheckBoxMenuItemUI$1.propertyChange(SubstanceCheckBoxMenuItemUI.java:114)
	at java.beans.PropertyChangeSupport.fire(PropertyChangeSupport.java:335)
	at java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:327)
	at java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:263)
	at java.awt.Component.firePropertyChange(Component.java:8428)
	at javax.swing.AbstractButton.setModel(AbstractButton.java:1781)
	at javax.swing.JMenuItem.setModel(JMenuItem.java:173)
	at games.strategy.triplea.ui.menubar.ViewMenu.addShowCommentLog(ViewMenu.java:98)
	at games.strategy.triplea.ui.menubar.ViewMenu.<init>(ViewMenu.java:88)
	at games.strategy.triplea.ui.menubar.TripleAMenuBar.<init>(TripleAMenuBar.java:31)
	at games.strategy.triplea.ui.TripleAFrame.<init>(TripleAFrame.java:235)
	at games.strategy.triplea.TripleA.lambda$startGame$363(TripleA.java:124)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:301)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at games.strategy.engine.framework.GameRunner$1.dispatchEvent(GameRunner.java:344)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

--- a/src/main/java/games/strategy/triplea/ui/TripleAFrame.java
+++ b/src/main/java/games/strategy/triplea/ui/TripleAFrame.java
@@ -231,7 +231,15 @@ public class TripleAFrame extends MainGameFrame {
     this.setCursor(uiContext.getCursor());
     editModeButtonModel = new JToggleButton.ToggleButtonModel();
     editModeButtonModel.setEnabled(false);
-
+    showCommentLogButtonModel = new JToggleButton.ToggleButtonModel();
+    showCommentLogButtonModel.setSelected(false);
+    showCommentLogButtonModel.addActionListener(e -> {
+      if (showCommentLogButtonModel.isSelected()) {
+        showCommentLog();
+      } else {
+        hideCommentLog();
+      }
+    });
     menu = new TripleAMenuBar(this);
     this.setJMenuBar(menu);
     final ImageScrollModel model = new ImageScrollModel();
@@ -273,15 +281,6 @@ public class TripleAFrame extends MainGameFrame {
     } else {
       mapAndChatPanel.add(mapPanel, BorderLayout.CENTER);
     }
-    showCommentLogButtonModel = new JToggleButton.ToggleButtonModel();
-    showCommentLogButtonModel.setSelected(false);
-    showCommentLogButtonModel.addActionListener(e -> {
-      if (showCommentLogButtonModel.isSelected()) {
-        showCommentLog();
-      } else {
-        hideCommentLog();
-      }
-    });
     gameMainPanel.setLayout(new BorderLayout());
     this.getContentPane().setLayout(new BorderLayout());
     this.getContentPane().add(gameMainPanel, BorderLayout.CENTER);

